.circleci/config.yml

```diff
@@ -103,6 +103,31 @@ jobs:
           name: Run renovate-config-validator command
           command: renovate-config-validator
 
+  # Check that all versions (backend, frontend, website and cookiecutter) are up-to-date
+  check-versions:
+    docker:
+      - image: circleci/buildpack-deps:stretch-scm
```

Comment:
I suggest you to not use the `stable` tag:
> stable - This was the previous name of the current tag. While it will still work for awhile, please use current instead.

Also current is suggest to have breaking changes. I choose to use the last available stable tag `2022.04` on Marsha and I will update it when we will have a future deprecation warning.

Reply:
ok so I'll change to : 
```suggestion
      - image: cimg/base:2022.04
```